src/constantinople_lab_to_nwb/schierek_embargo_2024/interfaces/schierek_embargo_2024_processedbehaviorinterface.py

```diff
@@ -107,7 +107,7 @@ def add_to_nwbfile(
             if column in data:
                 data[column] = list(np.array(data[column]).astype(bool))
 
-        columns_to_add = data.keys()
+        columns_to_add = list(data.keys())
```

Comment:
rollback unrelated change
```suggestion
        columns_to_add = data.keys()
```